src/components/transportation-icon/index.tsx

```diff
@@ -77,7 +87,15 @@ export function getTransportModeSvg(mode?: AnyMode) {
 
 const useStyles = StyleSheet.createThemeHook((theme) => ({
   transportationIcon: {
-    padding: 3,
-    borderRadius: theme.border.radius.small,
+    display: 'flex',
```

Comment:
Spinner opp for å sjekke ut dette nå etter lunsj. Men bare for ordens skyld, dette blir bakoverkompatible endringer som gjør at ikonene vises tilsvarende selv uten linjenummer? Så det skal i utgangspunktet ikke endre der transportation-icon blir brukt ellers?

Reply:
det blir noen tweaks i henhold til design, slik at at "boblene" blir litt høyere, og de får litt mer space rundt seg. med denne PR-en så er linjenummer en optional parameter, og funker uten, slik at de blir bakoverkompatible